pydoc.el

```diff
@@ -572,7 +578,7 @@ These are lines marked by `pydoc-example-code-leader-re'."
 
 (defun pydoc-latex-overlays-1 (limit)
   "Overlay images on \(eqn\) up to LIMIT."
-  (while (re-search-forward "\\\\([^ ]*?\\\\)" limit t)
+  (while (re-search-forward "\\\\([^�]*?\\\\)" limit t)
```

Comment:
what is happening with this and the changes below? are these just whitespace changes?

Reply:
Oops! I force pushed some new changes with all the fixes.